apps/backend/src/db/migrations/20250805171700_super_admin_seed.ts

```diff
@@ -0,0 +1,47 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	GROUPS: "groups",
+	USERS: "users",
+} as const;
+
+const GROUP_KEY = "admins";
+
+const ColumnName = {
+	EMAIL: "email",
+	ID: "id",
+	KEY: "key",
+} as const;
+
+type IdRecord = { id: number };
+
+const SUPERADMIN_EMAIL = "superadmin@mail.com";
+
+async function down(knex: Knex): Promise<void> {
+	await knex(TableName.USERS).where(ColumnName.EMAIL, SUPERADMIN_EMAIL).del();
+}
+
+async function up(knex: Knex): Promise<void> {
+	await knex.transaction(async (trx) => {
+		const group = await trx<IdRecord>(TableName.GROUPS)
+			.select(ColumnName.ID)
+			.where(ColumnName.KEY, GROUP_KEY)
+			.first();
+
+		if (!group || !group.id) {
+			throw new Error(`Group "${GROUP_KEY}" not found`);
+		}
+
+		await trx(TableName.USERS).insert({
+			email: SUPERADMIN_EMAIL,
+			first_name: "Super",
+			group_id: group.id,
+			last_name: "Admin",
```

Comment:
let's use something more close to real data
```suggestion
			first_name: "Anna",
			group_id: group.id,
			last_name: "Smith",
```